src/Lykke.Service.OperationsHistory.Mongo/OperationsHistoryRepository.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices.ComTypes;
+using System.Threading.Tasks;
+using Lykke.Service.OperationsHistory.Core;
+using MongoDB.Driver;
+using Lykke.Service.OperationsRepository.Contract;
+
+namespace Lykke.Service.OperationsHistory.Mongo
+{
+    public class OperationsHistoryRepository : IOperationsHistoryRepository

```

Comment:
Please take a look at https://github.com/LykkeCity/Lykke.Service.HFT/blob/master/src/Lykke.Service.HFT.MongoRepositories/MongoRepository.cs
may be we can use the common way for working with MongoDB / CosmosDB.